### PR TITLE
feat: rename active session name and the working dir

### PR DESCRIPTION
**Pull request by ariakh55**
This will add a feature to rename the active session along with its directory.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -171,13 +171,27 @@ pub(crate) fn set_up_tmux_env(repo: &Repository, repo_name: &str) -> Result<(),
...
+            let mut _args = splitted_args;
+            _args.remove(0);
+
+            _args
+        }
+
+        Err(_) => panic!("Failed to parse arguments"),
+    };
+    
+    execute_command("tmux", args)
```
Comment (ariakh55) [inline src/main.rs:194]: Also I have added another function to execute other commands

**Comment by jrmoulton** [comment]
I might just be missing it in the code but what is the benefit of adding this command to tmux sessionizer instead of just using tmux built in session rename functionality?

**Comment by ariakh55** [comment]
tmux session rename will only rename the session but with this feature not only the session but also the directory will be renamed
and it will automatically change directory of all the panes to the new renamed directory

The whole idea came from the time I used tmux-sessionzier to manage my git repositories and for some projects I decided to keep the project branches isolated from master.

**Comment by ariakh55** [comment]
This is a demo video showing how it works
[Screencast from 20.01.2024 10:05:49.webm](https://github.com/jrmoulton/tmux-sessionizer/assets/29955501/ad6f72ed-7b42-4871-86a7-998ff18f3cbe)

**Comment by jrmoulton** [comment]
Ahh interesting. Yeah I think this is in scope and would be useful. 

Can you update the readme with the updated output from the help command?

File: README.md [hunk 2]
```diff
@@ -43,6 +43,17 @@ Similar to `tms switch`, you can show other active windows in the current sessio
...
+### The `tms rename` command
+
+Using this command you can automatically rename the active session along with the directory name
+and the active directory inside all the panes in the active session will be changed to the renamed
+directory
+
+`tms rename <new_session_name>`
+
+`bind C-w command-prompt -p "Rename active session to: " "run-shell 'tms rename %1'"`.
+
```
Comment (ariakh55) [inline README.md:56]: I also added a sample bind example to it

**Pull request merged**